### PR TITLE
Add manifest for org.kde.kget

### PR DESCRIPTION
**Pull request by himanshu007-creator**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by himanshu007-creator** [comment]
bot, build org.kde.kruler

**Comment by hfiguiere** [comment]
What's the difference with #2741 ?

Also PR are supposed to be against the `new-pr` branch, not `master`.

File: org.kde.kget.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=wayland",
+    "--filesystem=host",
+    "--share=network"
+  ],
+
+  "modules": [
+    "boost.json",
+    "gpgme.json",
```
Comment (hfiguiere) [inline org.kde.kget.json:21]: Same as #2741 this is in the runtime

File: org.kde.kget.json [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+          "url": "https://invent.kde.org/kde/libktorrent.git",
+          "tag": "v21.08.1"
+        }
+      ]
+    },
+    {
+      "name": "kget",
+      "buildsystem": "cmake-ninja",
+      "sources": [
+        { "type": "git", "url": "https://invent.kde.org/kde/kget.git" }
```
Comment (hfiguiere) [inline org.kde.kget.json:60]: you need a commit / tag here.

File: org.kde.kget.json [hunk 3]
```diff
@@ -0,0 +1,64 @@
+{
+  "id": "org.kde.kget",
+  "branch": "master",
```
Comment (hfiguiere) [inline org.kde.kget.json:3]:
```suggestion
```

File: org.kde.kget.json [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15-21.08",
+  "sdk": "org.kde.Sdk",
+  "command": "kget",
+  "tags": ["nightly"],
+  "desktop-file-name-suffix": " (Nightly)",
```
Comment (hfiguiere) [inline org.kde.kget.json:9]:
```suggestion
```

File: org.kde.kget.json [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+  "rename-icon": "kget",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
```
Comment (hfiguiere) [inline org.kde.kget.json:13]:
```suggestion
    "--socket=fallback-x11",
```

File: org.kde.kget.json [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+    "--socket=wayland",
+    "--filesystem=host",
```
Comment (hfiguiere) [inline org.kde.kget.json:15]: I'm of the opinion this should only be `xdg-download`

File: org.kde.kget.json (hunk 1 above)
Comment (hfiguiere) [inline org.kde.kget.json:21]: both of these are missing.

**Review by hfiguiere — CHANGES_REQUESTED** [review]
This is a duplicate of #2741

**Comment by himanshu007-creator** [comment]
closing as PR is a duplicate of #2741. Thanks @hfiguiere

**Pull request closed without merging**